examples/error-handling.ts

```diff
@@ -0,0 +1,103 @@
+import * as express from 'express'
+import { NonEmptyString } from 'io-ts-types/lib/NonEmptyString'
+import * as H from '../src'
+import { toRequestHandler } from '../src/express'
+import { pipe } from 'fp-ts/lib/pipeable'
+
+//
+// model
+//
+
+interface User {
+  name: string
+}
+
+//
+// business logic
+//
+
+const UserNotFound = 'UserNotFound' as const
+
+const InvalidArguments = 'InvalidArguments' as const
+
+const JSONError = 'JSONError' as const
+
+type UserError = 'InvalidArguments' | 'UserNotFound' | 'JSONError'
```

Comment:
It wouldn't be better to have a single source of truth?
```suggestion
type UserError = typeof InvalidArguments | typeof UserNotFound | typeof JSONError
```